typescript-json-schema.ts

```diff
@@ -570,7 +570,7 @@ export class JsonSchemaGenerator {
         return undefined;
     }
 
-    private getDefinitionForProperty(prop: ts.Symbol, node: ts.Node) {
+    private getDefinitionForProperty(prop: ts.Symbol, node: ts.Node): Definition|null {
```

Comment:
```suggestion
    private getDefinitionForProperty(prop: ts.Symbol, node: ts.Node): Definition | null {
```

Reply:
> I think it's helpful.

Great!

> Can you find out why Travis tests are not running?

I'm not familiar with the Travis setup yet ...

By the way, would using a code formatter like Prettier be useful?  (I see that https://github.com/vega/ts-json-schema-generator also uses Prettier)

Reply:
FYI, I fixed this by pushing another commit before I realised that I could just click on the `Commit suggestion` button.